controller/src/main/java/io/pravega/controller/server/rpc/grpc/v1/ControllerServiceImpl.java

```diff
@@ -147,6 +149,16 @@ public void scale(ScaleRequest request, StreamObserver<ScaleResponse> responseOb
     }
 
     @Override
+    public void checkScale(ScaleStatusRequest request, StreamObserver<ScaleStatusResponse> responseObserver) {
+        log.info("check scale status called for stream {}/{}.", request.getStreamInfo().getScope(),
```

Comment:
all incoming calls in rpc are logging at info level. However, a case can be made to break the symmetry here because this could be called lot more than other apis.